README.md

```diff
@@ -22,7 +22,7 @@
 
 ## Introduction
 
-**nf-core/circrna** is a bioinformatics best-practice analysis pipeline for Quantification, miRNA target prediction and differential expression analysis of circular RNAs.
+**nf-core/circrna** is a bioinformatics best-practice analysis pipeline for BSJ detection, quantification, annotation and miRNA target prediction of circular RNAs.
```

Comment:
Perhaps put what the acronym BSJ stands for parentheses.

Reply:
Happens when you are too used to the acronyms - will clarify